frontend/src/components/Course/styled.ts

```diff
@@ -211,10 +211,11 @@ export const InfoContainer = styled(Box)({
     padding: '0px',
     flexDirection: 'row',
   },
-  height: 'fit-content',
+  height: '50px',
   display: 'flex',
-  alignItems: 'middle',
-  alignSelf: 'end',
+  alignItems: 'end !important',
+  alignSelf: 'end !important',
+  paddingBottom: '0px !important',
 });
```

Comment:
This changes create error in mobile design: https://prnt.sc/y7sTKNjF7Ew9.
You should apply the styles, that you've added only for theme.breakpoints.up('md'), in this case it will work correctly

Reply:
Ok!, I refused its feature

Reply:
Alright, so I had refused style from, 070 task, but i added additional styles for `up lg` resolution. Run pls on u computer this version